### PR TITLE
generate_binlog_position support tsl && update vendor

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

**Review by holys — COMMENTED** [review]
LGTM

**Comment by holys** [comment]
@GregoryIan  PTAL

File: generate_binlog_position/security.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 PingCAP, Inc.
```
Comment (IANTHEREAL) [inline generate_binlog_position/security.go]: put a pkg? other tools can use it

**Comment by IANTHEREAL** [comment]
Rest LGTM

**Pull request merged**